src/app/globals.css

```diff
@@ -1,56 +1,3 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
-
-:root {
-  --foreground-rgb: 0, 0, 0;
-  --background-start-rgb: 214, 219, 220;
-  --background-end-rgb: 255, 255, 255;
-}
-
-@media (prefers-color-scheme: dark) {
-  :root {
-    --foreground-rgb: 255, 255, 255;
-    --background-start-rgb: 0, 0, 0;
-    --background-end-rgb: 0, 0, 0;
-  }
-}
-
-body {
-  color: rgb(var(--foreground-rgb));
-  background: linear-gradient(
-      to bottom,
-      transparent,
-      rgb(var(--background-end-rgb))
-  ) rgb(var(--background-start-rgb));
-}
-
-@layer base {
-  h1 {
-    @apply text-3xl;
-  }
-  h2 {
-    @apply text-2xl;
-  }
-  h3 {
-    @apply text-xl;
-  }
-
-  ol, ul {
-    @apply pl-5;
-  }
-
-  ol {
-    @apply list-decimal;
-  }
-
-  ul {
-    @apply list-disc;
```

Comment:
Ten cały layer Base to style które dodałem jako defaultowe w poprzednim PR, bo bez nich wszystkie style dla elementów HTML są zerowane, nie ma wcięć dla list, a headery są tych samych wielkości.
Rozumiem, że radix theme dodaje swoje style do globalnych elementów żeby h1 nie wyglądał jak zwykły span? 😁